From 43cf5a5340b686a61aae9119cfa8fd2c8559913e Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 22 May 2015 11:32:41 +0800
Subject: [PATCH 1119/1594] MLK-10862-1 mxc IPUv3: common: Move the helper
 _ipu_is_smfc_chan() to ipu_prv.h

commit fa12f70fb32e52982d17bdc69fbdad0c232046f9 from
git://git.freescale.com/imx/linux-2.6-imx.git

The helper _ipu_is_smfc_chan() needs to be used outside ipu_common.c so move
it to ipu_prv.h.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/mxc/ipu3/ipu_common.c |    5 -----
 drivers/mxc/ipu3/ipu_prv.h    |    5 +++++
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/mxc/ipu3/ipu_common.c b/drivers/mxc/ipu3/ipu_common.c
index de262b9..e9df44e 100644
--- a/drivers/mxc/ipu3/ipu_common.c
+++ b/drivers/mxc/ipu3/ipu_common.c
@@ -86,11 +86,6 @@ static inline int _ipu_is_dmfc_chan(uint32_t dma_chan)
 	return ((dma_chan >= 23) && (dma_chan <= 29));
 }
 
-static inline int _ipu_is_smfc_chan(uint32_t dma_chan)
-{
-	return ((dma_chan >= 0) && (dma_chan <= 3));
-}
-
 static inline int _ipu_is_trb_chan(struct ipu_soc *ipu, uint32_t dma_chan)
 {
 	return (((dma_chan == 8) || (dma_chan == 9) ||
diff --git a/drivers/mxc/ipu3/ipu_prv.h b/drivers/mxc/ipu3/ipu_prv.h
index d288e99..3d5c293 100644
--- a/drivers/mxc/ipu3/ipu_prv.h
+++ b/drivers/mxc/ipu3/ipu_prv.h
@@ -164,6 +164,11 @@ enum ipu_dmfc_type {
 	DMFC_HIGH_RESOLUTION_ONLY_DP,
 };
 
+static inline int _ipu_is_smfc_chan(uint32_t dma_chan)
+{
+	return ((dma_chan >= 0) && (dma_chan <= 3));
+}
+
 static inline u32 ipu_cm_read(struct ipu_soc *ipu, unsigned offset)
 {
 	return readl(ipu->cm_reg + offset);
-- 
1.7.5.4

